src/react-apps/ux-editor/src/components/config/ConditionalRenderingComponent.tsx

```diff
@@ -175,10 +177,49 @@ class ConditionalRendering extends React.Component<IConditionalRenderingComponen
     this.props.deleteConnection(this.props.connectionId);
   }
 
+  public renderConditionalRenderingTargetComponentOption = (id: string): JSX.Element => {
+    return (
+      <option key={id} value={id}>
+        {this.props.formLayoutComponents[id].title + ' (id=' + id + ')'}
+      </option>
+    );
+  }
+
+  public renderCondtionalRenderingTargetContainerOptions = (id: string, baseContainer?: boolean): JSX.Element[] => {
+    let options: JSX.Element[] = [];
+    if (!this.props.order[id]) return options;
+    if (!baseContainer) {
+      options.push(
+        <option key={id} value={id}>
+          {'Container (id=' + id + ')'}
```

Comment:
Consider using ES6 Template Literal for better readability

{`Container (id=${id})`}

Reply:
Neat, thanks!

Reply:
``{ `Container (id=${id}` }`` 
(Correct markdown-format on Mats comment)